projects/core/src/store-finder/model/search-config.ts

```diff
@@ -1,4 +1,4 @@
-export interface SearchConfig {
+export interface StoreFinderSearchConfig {
   pageSize?: number;
   currentPage?: number;
   sort?: string;
```

Comment:
Not sure we still need this one. In `occ.models.ts`, we have `PaginationModel`.

Reply:
I'm not sure also and would prefer to not change the logic here in this task. Maybe we can create some additional refactoring issue,